packages/tools/examples/stackManipulationTools/index.ts

```diff
@@ -99,7 +99,7 @@ async function run() {
   toolGroup.addTool(WindowLevelTool.toolName);
   toolGroup.addTool(PanTool.toolName);
   toolGroup.addTool(ZoomTool.toolName);
-  toolGroup.addTool(StackScrollMouseWheelTool.toolName, { loop: true });
+  toolGroup.addTool(StackScrollMouseWheelTool.toolName, { loop: false });
```

Comment:
This seems unrelated - that is probably ok, but wasn't sure you intended to have this.

Reply:
Yeah the demo was feeling weird that is why i have it reverted